### PR TITLE
Do not convert /o\ into /O\

### PR DESCRIPTION
**Pull request by StyXman**
There is code that already check that a guy raising (at least) his right arm does not get big-headed (a.k.a uppercased). Just add the same for the other, poor guy complaining about an error.

### CODE REVIEW

File: requests/status_codes.py [hunk 1]
```diff
@@ -87,5 +87,5 @@
 for code, titles in _codes.items():
     for title in titles:
         setattr(codes, title, code)
-        if not title.startswith('\\'):
+        if not title.startswith('\\') and not title.strartswith('/'):
```
Comment (Lukasa) [inline requests/status_codes.py]: I don't think `strartswith` is the name of this method.

Comment (Lukasa) [reply requests/status_codes.py]: Additionally, you can write this as `if not title.startswith(('\\', '/'))`, which may be clearer.

**Comment by StyXman** [comment]
Yes, sorry, I just (w)hacked it in 3 minutes without proper testing. Shame on me. Let me see...

File: requests/status_codes.py [hunk 2]
```diff
@@ -87,5 +87,5 @@
 for code, titles in _codes.items():
     for title in titles:
         setattr(codes, title, code)
-        if not title.startswith('\\'):
+        if not if not title.startswith(('\\', '/')):
```
Comment (Lukasa) [inline requests/status_codes.py]: You have duplicated the `if not` here.

**Comment by Lukasa** [comment]
Cool, looks good now, thanks! :sparkles:

**Comment by StyXman** [comment]
\o/

**Pull request merged**